lib/recorder.js

```diff
@@ -1,7 +1,7 @@
 const debug = require('debug')('codeceptjs');
 const promiseRetry = require('promise-retry');
 
-const log = require('./output').log;
+const output = require('./output');
```

Comment:
I think `const { log } = require('./output');` will work and you don't need to change much lines of code.